### ISSUE TITLE
security vulnerabilities

### ISSUE THREAD

**Issue by dppatel99** [prometheus/alertmanager#3340]
Hi team,

In our vulnerability scan report there are 8 HIGH and 1 CRITICAL vulnerabilities in prometheus/alertmanager:v0.25.0

Component | Vulnerability | Severity | Installed Version | Latest Version | Description
| ---------- | ------------ | -------- | ----------------- | ---- | ---- |
| golang-runtime      | [CVE-2023-24538](https://nvd.nist.gov/vuln/detail/CVE-2023-24538) | CRITICAL | 1.19.4 | 1.20.3 | https://github.com/advisories/GHSA-v4m2-x4rp-hv22
|  golang-runtime  | [CVE-2023-24536](https://nvd.nist.gov/vuln/detail/CVE-2023-24536)   | HIGH | 1.19.4 | 1.20.3 | https://github.com/advisories/GHSA-9f7g-gqwh-jpf5
|  golang-runtime  | [CVE-2023-24534](https://nvd.nist.gov/vuln/detail/CVE-2023-24534)   | HIGH | 1.19.4 | 1.20.3 | https://github.com/advisories/GHSA-8v5j-pwr7-w5f8
|  golang-runtime | [CVE-2023-24537](https://nvd.nist.gov/vuln/detail/CVE-2023-24537)   | HIGH | 1.19.4 | 1.20.3  | https://github.com/advisories/GHSA-fp86-2355-v99r
|  golang-runtime | [CVE-2022-41722](https://nvd.nist.gov/vuln/detail/CVE-2022-41722)   | HIGH | 1.19.4 | 1.20.3  | https://github.com/advisories/GHSA-fp44-cj2j-3jhx
|  golang-runtime | [CVE-2022-41723](https://nvd.nist.gov/vuln/detail/CVE-2022-41723)   | HIGH | 1.19.4 | 1.20.3  | https://github.com/advisories/GHSA-vvpx-j8f3-3w6h
|  golang-runtime | [CVE-2022-41724](https://nvd.nist.gov/vuln/detail/CVE-2022-41724)  | HIGH | 1.19.4 | 1.20.3  | https://github.com/advisories/GHSA-89mw-w342-mqrr
|  golang-runtime | [CVE-2022-41725](https://nvd.nist.gov/vuln/detail/CVE-2022-41725)  | HIGH | 1.19.4 | 1.20.3  | https://github.com/advisories/GHSA-w4h2-22wh-m6jx
|  busybox | [CVE-2022-28391](https://nvd.nist.gov/vuln/detail/CVE-2022-28391)  | HIGH | 1.34.1 | 1.35.0  | https://github.com/advisories/GHSA-h8c3-8522-vxc6

My suggestion: Golang and busybox version update to 1.20.3 and 1.36.0 respectively would solve the problem.

**Comment by dppatel99** [comment]
Can we expect a fix in the upcoming release ?

**Comment by simonpasquier** [comment]
Yes these warnings should be fixed in the next Alertmanager release. No ETA though.

**Comment by dppatel99** [comment]
Thanks !! I will be waiting for the latest release

**Comment by ry4nz** [comment]
any ETA for the release?

**Comment by yevchuk-kostiantyn** [comment]
Hi @simonpasquier, we were wondering if it's now possible to share some ETA/plans for the next release?
Thanks in advance!

**Comment by TheMeier** [comment]
Please do not report raw vulnerability scanner results. They are prone to false positives and cause the Prometheus team toil in verifying. Please verify vulnerability reports and include specific details as to which components are directly exploitable.

https://prometheus.io/docs/operating/security/#automated-security-scanners

Also there have bneen several release in the meantime
